<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightSettingDialog</class>
 <widget class="QDialog" name="LightSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,3">
     <item row="0" column="0">
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="deletePushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="colorPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="3" colspan="2">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Position Coordinates</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="pXDoubleSpinBox">
          <property name="minimum">
           <double>-10000000000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="pYDoubleSpinBox">
          <property name="minimum">
           <double>-10000000000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="pZDoubleSpinBox">
          <property name="minimum">
           <double>-10000000000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>FocalPoint Coordinates</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="fXDoubleSpinBox">
          <property name="minimum">
           <double>-10000000000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="fYDoubleSpinBox">
          <property name="minimum">
           <double>-10000000000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="fZDoubleSpinBox">
          <property name="minimum">
           <double>-10000000000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
